src/book.py

```diff
@@ -188,36 +188,95 @@ def _read_coinbase(self, file_path: Path) -> None:
 
                 # Unused variables.
                 del eur_total
-                del remark
 
                 # Validate data.
                 assert operation
                 assert coin
                 assert change
-                assert eur_spot
 
-                self.append_operation(
-                    operation, utc_time, platform, change, coin, row, file_path
-                )
+                if operation == "Convert":
+                    # parse change + coin from remark which is in format "0,123 ETH to 0,456 BTC"
+                    remark_target = remark.split(' to ')[-1]
+                    remark_target_parts = remark_target.split(' ')
+                    _convert_change = remark_target_parts[0].replace(',', '.')
+                    convert_change = misc.xdecimal(_convert_change)
+                    convert_coin = remark_target_parts[1]
```

Comment:
The regex is implemented now. I initially decided against it because I mostly don't think regular expressions are the most readable and maintainable way but I think it turned out quite okay 😊